tests/tpubkey

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+# Copyright (C) 2022 Jakub Jelen <jjelen@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+
+source ${TESTSSRCDIR}/helpers.sh
+
+TSTOUT="${TMPPDIR}/testout"
+
+title PARA "Export RSA Public key to a file"
+ossl 'pkey -in $BASEURI -pubin -pubout -out ${TSTOUT}.pub'
+title LINE "Export Public key to a file (pub-uri)"
+ossl 'pkey -in $PUBURI -pubin -pubout -out ${TSTOUT}.pub'
+title LINE "Export Public key to a file (with pin)"
+ossl 'pkey -in $BASEURIWITHPIN -pubin -pubout -out ${TSTOUT}.pub'
```

Comment:
I wonder if we should move this to the tbasic section where we remove the pin from the configuration in order to be useful ..

Reply:
good point. Will move